css-view-transitions-2/Overview.bs

```diff
@@ -616,186 +563,391 @@ Prepend this to the [=Perform pending transition operations=] algorithm given a
 
-		1. Let |transition| be a new {{ViewTransition}} in |document|'s [=relevant Realm=],
-			whose [=ViewTransition/named elements=] is |inboundViewTransitionParams|'s [=view transition params/named elements=],
-			and [=ViewTransition/initial snapshot containing block size=] is |inboundViewTransitionParams|'s [=view transition params/initial snapshot containing block size=].
+		1. Otherwise, if |callbackOptions| is a {{StartViewTransitionOptions}}, then set |updateCallback| to |callbackOptions|'s {{StartViewTransitionOptions/update}}.
 
-		1. Set |document|'s [=active view transition=] to |transition|.
+		1. If |this|'s [=active view transition=] is not null and its [=outbound post-capture steps=] is not null,
+			then:
 
-		1. [=Resolve=] |transition|’s [=ViewTransition/update callback done promise=] with undefined.
+			1. Let |preSkippedTransition| be a new {{ViewTransition}} in |this|'s [=relevant realm=] whose [=ViewTransition/update callback=] is |updateCallback|.
 
-		1. Set |transition|’s [=ViewTransition/phase=] to "`update-callback-called`".
+			1. [=Skip the view transition|Skip=] |preSkippedTransition| with an "{{InvalidStateError}}" {{DOMException}}.
 
-		1. Set |transition|'s [=ViewTransition/active types=]'s [=DOMTokenList/token set=] to |resolvedRule|.
+			1. Return |preSkippedTransition|.
 
-		1. At any given time, the UA may decide to skip the inbound transition, e.g. after an [=implementation-defined=] timeout.
-			To do so, the UA should [=queue a global task=] on the [=DOM manipulation task source=] given |document|'s [=relevant global object=] to perform the following step:
-				If |transition|'s [=ViewTransition/phase=] is not "`done`", then [=skip the view transition=] |transition| with a "{{TimeoutError}}" {{DOMException}}.
+		1. Let |viewTransition| be the result of running the [=method steps=] for {{Document/startViewTransition(updateCallback)}} given |updateCallback|.
 
-		1. Return |transition|.
+		1. If |callbackOptions| is a {{StartViewTransitionOptions}}, set |viewTransition|'s [=ViewTransition/active types=] to a [=list/clone=] of {{StartViewTransitionOptions/types}} as a [=/set=].
+
+		1. Return |viewTransition|.
 	</div>
 
-### Updating the opt-in state for outbound view transitions ### {#update-opt-in-state-for-outbound}
+### Extending the {{ViewTransition}} interface ### {#view-transitions-extension}
 
-<div algorithm="update-opt-in-for-outbound">
-To <dfn>update the opt-in state for outbound transitions</dfn> for a {{Document}} |document|:
-	1. If |document| [=has been revealed=], and the result of [=resolve @view-transition rule|resolving the @view-transition rule=] is not "<code>skip transition</code>",
-		then set |document|'s [=can initiate outbound view transition=] to true.
-	1. Otherwise, set |document|'s [=can initiate outbound view transition=] to false.
+The {{ViewTransition}} interface is extended as follows:
+
+<xmp class='idl'>
+interface ViewTransitionTypeSet {
+	setlike<DOMString>;
```

Comment:
What is setlike? And why the indirection instead of using this directly as an attribute?